### PR TITLE
Remove Circular ingestion from legacy email

### PR DESCRIPTION
**Pull request by lpsinger**

### CODE REVIEW

**Comment by dakota002** [comment]
@lpsinger Tests are passing now, I don't know why they started failing all of a sudden, but this seems to have resolved the issue

**Review by lpsinger — COMMENTED** [review]
Looks good!

**Review by dakota002 — APPROVED** [review]

**Comment by lpsinger** [comment]
Oh no! Revert! We need to wait until December 31 to merge this.

**Pull request merged**
